instrumentation/active_job/test/test_helper.rb

```diff
@@ -25,6 +25,15 @@ def perform
   end
 end
 
+class RetryWithWaitJob < ActiveJob::Base
+  retry_on StandardError, wait: 0, attempts: 2
+
+  def perform
+    sleep(0.01)
```

Comment:
I initially tried setting `wait` on this job, but then the retry attempt was not running in the tests.  I think that makes sense as the retry would go back into the queue.  I don't love throwing a `sleep` in here, and I am open to suggestions.  This seemed like the simplest way to simulate the passage of time, for the `first_enqueued_at` tests, though.